### DISCUSSION TITLE
Reversing the order of selected publications

### DISCUSSION THREAD

**Discussion by sanghosuh** [alshedivat/al-folio#455] [Q&A]
How do we reverse the order of selected publications?
I am not sure how I should edit `{% bibliography -f papers -q @*[selected=true]* %}` in `/include/selected_publications.html`
I would appreciate any suggestion on this. Thank you!

**Comment by alshedivat** [comment]
you should be able to override the order of selected publications by adding the following to the header of `_pages/about.md`:
```
---
scholar:
  sort_by: year
  order: ascending  # can be either "ascending" or "descending"
---
```
source: https://github.com/inukshuk/jekyll-scholar/issues/299

**Reply by sanghosuh** [reply]
Thank you. I tried, but this doesn't seem to work. Do you have any other suggestions?

**Reply by alshedivat** [reply]
ah, I think there was a typo in the original solution in jekyll-scholar issue I linked above. try changing jekyll: to scholar: (as in the updated code snippet above).

**Reply by sanghosuh** [reply]
Sorry, that doesn't seem to work either. Your suggestion was to just add that to the header, right? That is, I didn't have to change anything in `/include/selected_publications.html`.

**Reply by alshedivat** [reply]
hmm, that's odd. I've just tested it on master and it seems to work. you can test it on a clean version of the repo. just clone it to some temporary folder and do the following 2 edits:

1. add `selected={true},` to the first reference in `_bibliography/papers.bib`. run `bundle exec jekyll serve` locally to make sure there are now two selected publications on the about page (with publication dates 1935 and 1956).
2. add the `scholar:` section from above to the preamble (ie, stuff in between `---` at the top of the file) in `_pages/about.md`.

without step 2, the two selected reference should be in descending order (default); after step 2, the ordering should change.
